product-service/src/handlers/getProductsList.ts

```diff
@@ -0,0 +1,23 @@
+import 'source-map-support/register';
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import { getAllProducts } from '../service/productService';
+
+export const getProductsList: APIGatewayProxyHandler = async () => {
+  try {
+    const products = await getAllProducts();
+
+    return {
+      statusCode: 200,
+      body: products,
+      headers: {
+        'Access-Control-Allow-Origin': '*',
+        'Access-Control-Allow-Credentials': true,
+      },
+    };
+  } catch (err) {
+    return {
+      statusCode: err,
```

Comment:
statusCode: should be a Number, But in your code, it is an Error with an empty message - [15 line](https://github.com/Raerka/nodejs-aws-be/blob/task3_rsschool/product-service/src/service/productService.ts)

Reply:
Fixed. Thank you